test/testserver/testserver.go

```diff
@@ -30,7 +30,11 @@ func (ts *TestServer) stub(l logging.SimpleLogger) http.HandlerFunc {
 				l.Logf("responding with body from %s", response.Fixture)
 				ts.Responses[r.URL.Path] = res[1:]
 				w.WriteHeader(response.StatusCode)
-				w.Write(response.Body)
+				_, err := w.Write(response.Body)
+				if err != nil {
+					http.Error(w, "write error", http.StatusInternalServerError)
```

Comment:
Perhaps it would be good to show the actual `err` ?

Reply:
This shouldn't really happen. If it does, not sure if we want to hand that information to the user. By the way, this is just a test server so this shouldn't really matter ;)